asmcomp/arm64/emit.mlp

```diff
@@ -636,7 +636,7 @@ let emit_instr i =
     | Lop(Iextcall { func; alloc = false; label_after = _; }) ->
         `	bl	{emit_symbol func}\n`
     | Lop(Iextcall { func; alloc = true; label_after; }) ->
-        emit_load_symbol_addr reg_x15 func;
+        emit_load_symbol_addr reg_x18 func;
```

Comment:
Here, we need a register that is not used for argument passing (so, not x0..x7), and not callee-save, but preserved by dynamic linking stubs.  We have a winner: x8 !